_docs/style/iconography.md

```diff
@@ -175,25 +175,49 @@ Rackspace US, Inc. retains copyright and/or applicable rights to all text and gr
 
 
 {% column right:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+// Do not use the embed tag on this image when we refactor SVGs
```

Comment:
Switch to `<!--` and `-->` for comments

![screen shot 2018-04-04 at 4 14 40 pm](https://user-images.githubusercontent.com/1538692/38335257-5e7d4b5c-3823-11e8-8221-355e027c716b.png)

Reply:
ow gosh that was silly of me. updating.